src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_params.py

```diff
@@ -27,12 +27,35 @@ def load_arguments(self, _):
         c.argument('enable_non_ssl_port', action='store_true')
         c.argument('shard_count', type=int)
 
+    with self.argument_context('redis import') as c:
+        c.argument('files', help='SAS url for blobs that needs to be imported', nargs='+')
+        c.argument('format', options_list=['--file-format'], help='Format of the blob (Example: rdb)')
+
     with self.argument_context('redis import-method') as c:
-        c.argument('files', nargs='+')
+        c.argument('files', help='SAS url for blobs that needs to be imported', nargs='+')
+        c.argument('file-format', help='Format of the blob (Example: rdb)')
+
+    with self.argument_context('redis export') as c:
+        c.argument('container', help='SAS url for container where data needs to be exported to')
+        c.argument('prefix', help='Prefix to use for exported files')
+        c.argument('file-format', help='Format of the blob (Example: rdb)')
 
     with self.argument_context('redis patch-schedule set') as c:
-        c.argument('schedule_entries', type=ScheduleEntryList)
+        c.argument('schedule_entries', help="List of Patch schedule entries. Example Value:[{\"dayOfWeek\":\"Monday\",\"startHourUtc\":\"00\",\"maintenanceWindow\":\"PT5H\"}]", type=ScheduleEntryList)
```

Comment:
This is a huge anti-pattern. Accepting a JSON string for a parameter is really bad UX.  I realize it is already in the CLI, but consider revising this command group.